composer.json

```diff
@@ -14,10 +14,11 @@
         }
     ],
     "require": {
-        "composer/installers": "*",
-        "silverstripe/framework": "^4.0",
-        "silverstripe/vendor-plugin": "^1"
+        "php": "^8.1",
+        "silverstripe/framework": "^5"
     },
+    "minimum-stability": "dev",
```

Comment:
Is this needed? Should be enough in the project's root composer.

Reply:
it is needed while CMS 5 is not stable yet. 

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires silverstripe/framework ^5, found silverstripe/framework[5.0.0-alpha1, ..., 5.x-dev] but it does not match your minimum-stability.

```

Reply:
bump. 

`minimum-stability` can be removed now

Reply:
Though, actually, it currently seems to work in both SilverStripe 4 and 5, so it might be a better choice to require `"silverstripe/framework": "^4.0 || ^5.0",` as long as we don't have any breaking changes for 4.

Reply:
And require PHP ^7.4 || ^8.0

Reply:
PR updated!

Reply:
excellent, thank you. But `prefer-stable` and `config` are root-only properties, they should not be added to this repo

Reply:
Also, should we actually remove `silverstripe/vendor-plugin`, is that not needed anymore?  
I'd recommend requiring `"silverstripe/vendor-plugin": "^2"`

Reply:
oh, sorry, it should be `"silverstripe/vendor-plugin": "^1 || ^2"` to be ss4 and ss5 compatible.